docs/python_support/python_support.md

```diff
@@ -0,0 +1,333 @@
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+make docker_build MEDIAPIPE_DISABLE=0 PYTHON_DISABLE=0 OV_USE_BINARY=1 RUN_TESTS=0
+cd ..
+```
+Resulting Docker image can be extended with additional layers just as the public one.
+
+## Python Servable
+
+When deploying Python servable, Model Server expects Python file with an `OvmsPythonModel` class implemented:
+
+```python
+class OvmsPythonModel:
+
+    def initialize(self, kwargs):
+        """
+        `initialize` is called when model server loads graph definition. It allows to initialize and maintain state between subsequent execute() calls and even graph instances. 
+        For gRPC unary, graphs are recreated per request. 
+        For gRPC streaming, there can be multiple graph instances existing at the same time. 
+        OvmsPythonModel object is initialized with this method and then shared between all graph instances. Implementing this function is required (should it be?).
+
+        Parameters:
+        -----------
+        kwargs : dict
+            Available arguments:
+            * node_name: string
+                Name of the node in the graph
+            * input_streams: list of strings
+                List of input stream names defined for the node in graph configuration
+            * output_streams: list of strings
+                List of output stream names defined for the node in graph configuration
+        -----------
+        """
+        print("Running initialize...")
+
+    def execute(self, inputs):
+        """
+        `execute` is called in `Process` method of PythonExecutorCalculator which in turn is called by Mediapipe framework. How Mediapipe calls `Process` method for the node depends on the configuration and the two configurations supported by PythonExecutorCalculator are:
+        
+        * Regular: `execute` runs every time the node receives inputs. Produces one set of outputs per one set of inputs. For unary endpoints it's the only possible configuration.
+
+        * Generative: `execute` runs multiple times for single inputs set. Produces multiple sets of outputs over time per single set of inputs. Works only with streaming endpoints. 
+
+        Implemeting this function is required.
+
+        Parameters:
+        -----------
+        * inputs: list of pyovms.Tensor
+        -----------
+        
+        Returns: list of pyovms.Tensor or generator
+        """
+        ...
+        return outputs
+
+    def finalize(self):
+        """
+        `finalize` is called when model server unloads graph definition. It allows to perform any cleanup actions before the graph is removed. Implementing this function is optional.
+        """
+        print("Running finalize...")
+```
+
+### initialize
+
+`initialize` is called when model server loads graph definition. It allows to initialize and maintain state between subsequent `execute` calls and even graph instances.
+
+For gRPC unary, graphs are recreated per request.
```

Comment:
Isnt it duplicated of the snippet above?

Reply:
Yes, it's supposed to have more details and better visual.